_posts/2018-02-22-totw-93.md

```diff
@@ -109,7 +109,7 @@ bool SaferMemCpy(absl::Span<T> dest, absl::Span<const T> src) {
   if (src.size() > dest.size()) {
     return false;
   }
-  memcpy(dest.data(), src.data(), src.size());
+  memcpy(dest.data(), src.data(), src.size_t);
```

Comment:
memcpy(dest.data(), src.data(), src.size() * sizeof(T));

and update the description